src/main/java/company/Company.java

```diff
@@ -0,0 +1,42 @@
+package rybka.company;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Company {
+    private List<Employee> employeeList = new ArrayList<>();
+    private Random random = new Random();
+
+    public Company(List<Employee> employeeList) {
```

Comment:
great!

Reply:
but why didn't u use isHired as the field in Employee class

Reply:
I thought it belongs to Company logic. But if it is not good, I will change it.